core-bundle/src/Slug/Slug.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Slug;
+
+use Ausi\SlugGenerator\SlugGenerator;
+use Contao\CoreBundle\Framework\ContaoFrameworkInterface;
+use Contao\PageModel;
+use Contao\StringUtil;
+
+class Slug
+{
+    /**
+     * @var SlugGenerator
+     */
+    private $slugGenerator;
+
+    /**
+     * @var ContaoFrameworkInterface
+     */
+    private $framework;
+
+    public function __construct(SlugGenerator $slugGenerator, ContaoFrameworkInterface $framework)
```

Comment:
We should probably use an interface here instead of `SlugGenerator`?

Should I add an interface (ausi/slug-generator#11) and release a new verion of the slug generator before we merge this?

Reply:
You can if you want. I doubt that many people will exchange the implementation, but using interfaces (contracts) instead of concrete classes is always a good idea.

Reply:
Done.